packages/react/src/components/ComboButton/index.js

```diff
@@ -155,6 +166,19 @@ ComboButton.propTypes = {
    */
   label: PropTypes.string.isRequired,
 
+  /**
+   * Specify how the menu should align with the button element
+   */
+  menuAlignment: PropTypes.oneOf([
+    'top',
+    'top-left',
+    'top-right',
+
+    'bottom',
+    'bottom-left',
+    'bottom-right',
+  ]),
```

Comment:
In #14654 we're working on refactoring popover placement options to unify against a naming scheme that aligns with logical properties. This is also what floating-ui uses, which Menu will eventually use at some point. 

```suggestion
  menuAlignment: PropTypes.oneOf([
    'top',
    'top-start',
    'top-end',

    'bottom',
    'bottom-start',
    'bottom-end',
  ]),
```

This will also impact your updates to the docs and such.